__tests__/app.test.js

```diff
@@ -115,3 +115,53 @@ describe("Get /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("status: 200 - returns comment object that corresponds with review_id stated by clients request", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews).toBeInstanceOf(Array);
+        expect(body.reviews.length).toBeGreaterThan(0);
```

Comment:
Since this is test data, we just check the exact length

Reply:
cool I'll sort that out